python-black.el

```diff
@@ -119,6 +119,22 @@ DISPLAY-ERRORS is non-nil, shows a buffer if the formatting fails."
   (and (= (point-min) beg)
        (= (point-max) end)))
 
+(defun python-black-block ()
+  "Reformats the current org-mode source block."
+  (interactive)
+  (save-mark-and-excursion
+    (when (org-in-block-p '("src" "example"))
+      (let* ((element (org-element-at-point))
+             (src-block-not-empty
+              (string-match
+               "[^\\s\\n]+"
+               (org-element-property :value element)))
+             (area (org-src--contents-area element)))
+        (when src-block-not-empty
+          (python-black-region
```

Comment:
i think it would be better to have this consistent with other helpers, which take+pass `&optional display-errors`, see e.g. `python-black-region` and `python-black-statement`

Reply:
taken into account